bedrock/mozorg/templates/mozorg/contact/communities/czech-republic.html

```diff
@@ -10,20 +10,32 @@
   <div id="entry-container">
     <section class="entry entry-community" id="czech-republic" data-tab="communities">
       <div class="card">
+        <img class="logo" src="{{ static('img/contact/logo-czech.png') }}" alt="Mozilla.cz">
         <h2>{{ _('Czech Republic') }}</h2>
 
         <ul class="extra">
-          <li><a href="http://www.mozilla.cz/" class="website">www.mozilla.cz</a></li>
+          <li><a href="https://www.mozilla.cz/" class="website">www.mozilla.cz</a></li>
+          <li><a href="https://www.mozilla.cz/podpora/" class="book">Support</a></li>
           <li><a href="mailto:info@mozilla.cz" class="email">info@mozilla.cz</a></li>
           <li><a href="https://twitter.com/MozillaCZ" class="twitter">@MozillaCZ</a></li>
-          <li><a href="https://www.facebook.com/Mozillacz" class="facebook">Facebook</a></li>
-          <li><a href="irc://irc.mozilla.org/mozilla-sk" class="irc">IRC: #mozilla-sk</a></li>
+          <li><a href="https://www.facebook.com/MozillaCZ" class="facebook">Facebook</a></li>
+          <li><a href="https://t.me/MozillaCZ" class="irc">Telegram: @MozillaCZ</a></li>
         </ul>
 
         <figure class="feature-img">
           <img src="//api.tiles.mapbox.com/v4/{{ settings.MAPBOX_TOKEN }}/15.20,49.72,6/380x250.png?access_token={{ settings.MAPBOX_ACCESS_TOKEN }}" alt="">
         </figure>
       </div>
+
+      <section class="gallery">
+        <h3>{{ _('Photo gallery') }}</h3>
+        <ul class="gallery-list">
+          <li class="gallery-image"><a href="{{ static('img/contact/pictures/czech-01.jpg') }}"><img src="{{ static('img/contact/pictures/czech-01-th.jpg') }}" alt="1"></a></li>
```

Comment:
These images are 404

Reply:
Sorry, missed the zero in the file names. Fixed now.